backend/src/ee/routes/v1/certificate-authority-crl-router.ts

```diff
@@ -1,86 +1,31 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
I noticed that the CRL distribution points for most certificates in the wild have a `.crl` extension.

Might be something we'll have to do/follow as well. I assume this is so that upon fetching, it gets saved as a `.crl` file 
<img width="612" alt="image" src="https://github.com/user-attachments/assets/438cfc97-f8dd-4064-bd14-8f45b8973df7">
<img width="686" alt="image" src="https://github.com/user-attachments/assets/38caf5d6-6f62-4ead-a09c-bd5115da802e">
<img width="559" alt="image" src="https://github.com/user-attachments/assets/54020b9c-57ad-4c90-846d-33a511569eee">

Reply:
I think it's fine to omit `.crl` part so long as the endpoint properly downloads the CRL according to the RFC; I've tested the current with `openssl` and it worked well.

Reply:
gotcha!